inspirehep/modules/orcid/tasks.py

```diff
@@ -85,7 +85,8 @@ def delete_from_orcid(sender, api=None):
         raw_user = UserIdentity.query.filter_by(
             id=record.orcid, method='orcid').first()
         user = RemoteAccount.query.filter_by(user_id=raw_user.id_user).first()
-        token = user.tokens[0].access_token
+        token = RemoteToken.query.filter_by(
+            id_remote_account=user.id).first().access_token
```

Comment:
`id_remote_account` seems to be the id of a remote account (https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/models.py#L124)

Is it the same as the user id?